package.json

```diff
@@ -5,8 +5,8 @@
   "version": "2.1.2",
   "author": "Evert Timberg <evert.timberg@gmail.com>",
   "license": "MIT",
-  "main": "dist/chartjs-plugin-annotation.js",
-  "module": "dist/chartjs-plugin-annotation.esm.js",
```

Comment:
This is wrong, the main needs to point to a commonjs build and module to esm build for older node support.
The for some other platforms (I think at least vite), we need to have the `exports` section.

And from that thought, maybe we should add another integration test with different node versions.
Should also have an integration test for both flavors of node, module and non-module package.
for example: https://github.com/kurkle/chartjs-chart-matrix/tree/main/test/integration

Reply:
OK, now it's clear!! I think we should change treemap chart as well!

> And from that thought, maybe we should add another integration test with different node versions.
> Should also have an integration test for both flavors of node, module and non-module package.
> for example: https://github.com/kurkle/chartjs-chart-matrix/tree/main/test/integration

about integration test, I think I'll need more time to have a look, changing the current PR. hopefully it's ok for you

Reply:
@kurkle I have a question about rollup. In matrix, you create .cjs with `format: 'umd'`. Instead, in Chart.js, `format: 'commonjs'` is used. Why is there this difference?

Reply:
Done!

Reply:
> @kurkle I have a question about rollup. In matrix, you create .cjs with `format: 'umd'`. Instead, in Chart.js, `format: 'commonjs'` is used. Why is there this difference?

umd works for commonjs and amd, so its a bit more generic. it adds a little wrapper to the file, so its able to work on both cases. not sure how much amd is used or if it is needed, but also don't see why not to support it.

Reply:
The some of the entry points are renamed so its a breaking change. I would not wait with it too long though, so maybe soon after 2.2 go for 3?

Or could just move all the 2.2 stuff to 3 instead and publish all at once.

Reply:
> The some of the entry points are renamed so its a breaking change. I would not wait with it too long though, so maybe soon after 2.2 go for 3?
> 
> Or could just move all the 2.2 stuff to 3 instead and publish all at once.

I have changed rollup config to create `chartjs-plugin-annotation.js` as well. From user perspective, I don't see breaking changes related to the renamed file now. 
I would like to go to next version anyway, including it. 
@kurkle just a hint: do you think it's better to go to version 3 or publish 2.2 with this one (remove breaking change, if there isn't)? For me no problem, just a matter to change the bump PR and some milestones, already assigned, before publishing.